aboutme.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio Liliya Voevodina.html</title>
+        <link href="css/normalize.css" rel="stylesheet" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="css/stylesheet.css" rel="stylesheet" type="text/css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div class="aboutme-hometop">
+            <nav>
+                <img class="aboutme-mypicture" src="images/IMG_9169.jpg" alt="Liliya">
+                <a href="#">Contacts</a>
+                <a href="#">Portfolio</a>
+                <a href="aboutme.html">About me</a>
+                <a href="homepage.html">Home</a>
+            </nav>
+            <div class="aboutme">
+            <h3>
+                I wish i was a developer...but i am not.
+            </h3>
+            <h4>
+                <p>
```

Comment:
Why not have <p> on its own, rather than nested inside an h4? We can then style it with CSS to have the font-size and weight that we want.

Reply:
I was thinking of using more p tags on other pages so have to use h to CSS them separetely.